mmv1/products/compute/SecurityPolicyRule.yaml

```diff
@@ -286,74 +287,75 @@ properties:
-      - 'network_match.destIpRanges'
-      - 'network_match.ipProtocols'
-      - 'network_match.srcPorts'
-      - 'network_match.destPorts'
-      - 'network_match.srcRegionCodes'
-      - 'network_match.srcAsns'
-    properties:
-      - !ruby/object:Api::Type::Array
-        name: 'userDefinedFields'
-        description: |
-          User-defined fields. Each element names a defined field and lists the matching values for that field.
-        item_type: !ruby/object:Api::Type::NestedObject
-          properties:
-            - !ruby/object:Api::Type::String
-              name: 'name'
-              description: |
-                Name of the user-defined field, as given in the definition.
-            - !ruby/object:Api::Type::Array
-              name: 'values'
-              description: |
-                Matching values of the field. Each element can be a 32-bit unsigned decimal or hexadecimal (starting with "0x") number (e.g. "64") or range (e.g. "0x400-0x7ff").
-              item_type: Api::Type::String
-      - !ruby/object:Api::Type::Array
-        name: 'srcIpRanges'
-        description: |
-          Source IPv4/IPv6 addresses or CIDR prefixes, in standard text format.
-        item_type: Api::Type::String
-      - !ruby/object:Api::Type::Array
-        name: 'destIpRanges'
-        description: |
-          Destination IPv4/IPv6 addresses or CIDR prefixes, in standard text format.
-        item_type: Api::Type::String
-      - !ruby/object:Api::Type::Array
-        name: 'ipProtocols'
-        description: |
-          IPv4 protocol / IPv6 next header (after extension headers). Each element can be an 8-bit unsigned decimal number (e.g. "6"), range (e.g. "253-254"), or one of the following protocol names: "tcp", "udp", "icmp", "esp", "ah", "ipip", or "sctp".
-        item_type: Api::Type::String
-      - !ruby/object:Api::Type::Array
-        name: 'srcPorts'
-        description: |
-          Source port numbers for TCP/UDP/SCTP. Each element can be a 16-bit unsigned decimal number (e.g. "80") or range (e.g. "0-1023").
-        item_type: Api::Type::String
-      - !ruby/object:Api::Type::Array
-        name: 'destPorts'
-        description: |
-          Destination port numbers for TCP/UDP/SCTP. Each element can be a 16-bit unsigned decimal number (e.g. "80") or range (e.g. "0-1023").
-        item_type: Api::Type::String
-      - !ruby/object:Api::Type::Array
-        name: 'srcRegionCodes'
-        description: |
-          Two-letter ISO 3166-1 alpha-2 country code associated with the source IP address.
-        item_type: Api::Type::String
-      - !ruby/object:Api::Type::Array
-        name: 'srcAsns'
-        description: |
-          BGP Autonomous System Number associated with the source IP address.
-        item_type: Api::Type::Integer
+  # These fields are used for security policies of the type CLOUD_ARMOR_NETWORK, this type is not supported by the global security policy resource.
```

Comment:
If the following fields won't be supported by the global security policy resource, I think we can just remove them.